defaults/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
 # Whether or not to install Python 2 alongside Python 3
 python_install_python2: false
+
+# Whether or not to install the Python development dependencies
+python_install_development_dependencies: false
```

Comment:
This variable should be alphabetically before `python_install_python2`.

Reply:
Please see commit d3d9731.